toolkit/docs/formats/imageconfig.md

```diff
@@ -194,6 +194,29 @@ A sample PackageLists entry pointing to three files containing package lists:
 ],
 ```
 
+### Disabling Documentation and Locales
+
+For size constrained images it may be desirable to omit documentation and non-default locales from an image.
+
+``` json
+"DisableRpmDocs": true,
+"OverrideRpmLocales": "NONE",
+```
+
+`DisableRpmDocs` will configure the image to never install the documentation. Selecting `OverrideRpmLocales=NONE` will omit all locales from the image.
+
+These customizations are encoded into `/usr/lib/rpm/macros.d/macros.installercustomizations_*` files on the final system.
+
+#### Custom Locales
+
+A specific locale string may also be set using:
+
+``` json
+"OverrideRpmLocales": "en:fr:es"
+```
+
+This may be any value compatible with the `%_install_langs` rpm macro.
```

Comment:
Could you also add a link to the RPM docs explaining how this configuration works, so it's easier for people to learn and understand what are their options and how to use this config?

Reply:
Best I can find are some tangentially related fedora planning docs and a few mailing list posts. No official documents that I could find unfortunately.